build/webpack.client.base.conf.js

```diff
@@ -17,7 +17,6 @@ module.exports = merge(baseWebpackConfig, {
 			chunks: 'all',
 			maxAsyncRequests: Infinity, // default is 6
 			cacheGroups: {
-				default: false,
```

Comment:
This is causing common modules to be duplicated across chunks instead of being added to a shared chunk, greatly increasing the initial payload sent to users.